src/extensions/default/Inline3DParametersEditor/main.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * File based on InlineColorEditor/main.js
+ */
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var EditorManager           = brackets.getModule("editor/EditorManager"),
+        ExtensionUtils          = brackets.getModule("utils/ExtensionUtils"),
+        HTMLUtils               = brackets.getModule("language/HTMLUtils"),
+        Inline3dParametersUtils = require("Parameters3DUtils"),
+        Inline3DParameterEditor = require("Inline3DParameterEditor").Inline3DParameterEditor;
+
+    /**
+     * Prepare hostEditor for an Inline3DParameterEditor at pos if possible. Return
+     * editor context if so; otherwise null.
+     *
+     * @param {Editor} hostEditor
+     * @param {{line:Number, ch:Number}} pos
+     * @return {?{parameter:String, marker:TextMarker}}
+     */
+    function prepareParametersForProvider(hostEditor, pos) {
+        var ParameterRegex, tagStart, cursorLine, match, sel, start, end, endPos, marker, tagInfo;
+
+        sel = hostEditor.getSelection();
+        if (sel.start.line !== sel.end.line) {
+            return null;
+        }
+        var tagInfo = HTMLUtils.getTagInfo(hostEditor, sel.start);
+        var tag = tagInfo.tagName;
+        tagStart =Inline3dParametersUtils.TAG_START;
+        if(!tag.substr(0, 2) === tagStart) {
+            return null;
+        }
+
+        ParameterRegex = new RegExp(Inline3dParametersUtils.PARAMETERS_3D_REGEX);
```

Comment:
Found the issue and fixed it [here](https://github.com/mozilla/brackets/pull/651/commits/64bd6502236b0a9630a4a594a7e2200ce06ac932)